lib/src/git.dart

```diff
@@ -57,7 +57,7 @@ Future<List<String>> run(List<String> args,
   log.muteProgress();
   try {
     var result = await runProcess(command, args,
-        workingDir: workingDir, environment: environment);
+        environment: {...?environment, 'LANG': 'en_GB'});
```

Comment:
This should also be changed in the 'runSync' function

Reply:
Done